ui/lib/pki/addon/components/page/pki-issuer-details.hbs

```diff
@@ -0,0 +1,77 @@
+<Toolbar>
+  <ToolbarActions>
+    {{#if @canRotate}}
+      <ToolbarLink @route="issuers.generate-root" @type="rotate-cw" @issuer={{@issuer.id}} data-test-pki-issuer-rotate-root>
+        Rotate this root
+      </ToolbarLink>
+    {{/if}}
+    {{#if @canCrossSign}}
+      <ToolbarLink
+        @route="issuers.issuer.cross-sign"
+        @type="pen-tool"
+        @issuer={{@issuer.id}}
+        data-test-pki-issuer-cross-sign
+      >
+        Cross-sign Issuer
+      </ToolbarLink>
+    {{/if}}
+    {{#if @canSignIntermediate}}
+      <ToolbarLink @route="issuers.issuer.sign" @type="pen-tool" @issuer={{@issuer.id}} data-test-pki-issuer-sign-int>
+        Sign Intermediate
+      </ToolbarLink>
+    {{/if}}
+    <DownloadButton
+      class="toolbar-link"
+      @filename={{@issuer.id}}
+      @data={{@issuer.certificate}}
+      @extension="pem"
+      data-test-issuer-download
+    >
+      Download
+      <Chevron @direction="down" @isButton={{true}} />
+    </DownloadButton>
+    {{#if @canConfigure}}
+      <ToolbarLink @route="issuers.issuer.edit" @issuer={{@issuer.id}} data-test-pki-issuer-configure>
+        Configure
+      </ToolbarLink>
+    {{/if}}
+  </ToolbarActions>
+</Toolbar>
+
+<main data-test-issuer-details>
```

Comment:
I haven't seen this before! Or noticed it - what does `<main>` determine?

Reply:
Main is [HTML-semantics](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) for the main content for the page. It's a little nice-to-have for accessibility purposes (see [Landmark](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#landmark))

Reply:
Rad! Thank you for explaining